src/Fantomas/CodePrinter.fs

```diff
@@ -1637,6 +1640,17 @@ and genExpr astContext synExpr ctx =
         | Paren (_, ILEmbedded r, _) ->
             // Just write out original code inside (# ... #)
             fun ctx -> !- (defaultArg (lookup r ctx) "") ctx
+
+        | Paren (lpr, SynExpr.Ident ident, rpr) when astContext.IsInsideIfThenElseExpression ->
```

Comment:
I prefer this check to happen inside the `Elif` match clause.
`genExpr` is called all over the place.